typescript/sdk/src/core/schemas.ts

```diff
@@ -3,12 +3,15 @@ import { z } from 'zod';
 import { ProxyFactoryFactoriesSchema } from '../deploy/schemas.js';
 import { HookConfigSchema } from '../hook/schemas.js';
 import { IsmConfigSchema } from '../ism/schemas.js';
-import { OwnableSchema } from '../schemas.js';
+import { DeployedOwnableSchema, OwnableSchema } from '../schemas.js';
 
 export const CoreConfigSchema = OwnableSchema.extend({
   defaultIsm: IsmConfigSchema,
   defaultHook: HookConfigSchema,
   requiredHook: HookConfigSchema,
+  // This field is set as optional because the old core config
+  // did not have it and we want to maintain backward compatibility
+  proxyAdmin: DeployedOwnableSchema.optional(),
```

Comment:
define "old core config" - out of 140ish chains with a `mailbox` in registry, 134 have a `proxyAdmin`

Reply:
For example, if someone uses an old config file where the `proxyAdmin` field was not defined because it was generated with an old version of the CLI, Example:
- Deploy core contracts with old cli
- Update the CLI to the new version that adds this field
- Try to read the core deployment config. If the fields is set as required this will fail

Reply:
I see - wonder if it's ok to just make it a breaking change that we demand the `proxyAdmin` now and if an old version doesn't have it then that's on the user

wdyt @ltyu?

Reply:
Technically it shouldn't be a problem because worst case scenario the user can just `core read` to 'update' the config to the latest version but it is an inconvenience anyways because the first time the CLI will fail and the fix is not that obvious. deferring to @ltyu for his thoughts. Maybe we should consider versioning configs in the future to avoid breaking changes and allows conversion to an old config file to a new one